### PR TITLE
CI for NNPOps

### PR DESCRIPTION
**Pull request by raimis**
- [x] Install the decencies
- [x] Build the library
- [x] Run the tests (CPU only)

### CODE REVIEW

**Comment by raimis** [comment]
This was tested on a fork: https://github.com/raimis/NNPOps/actions/runs/1304947584

**Comment by raimis** [comment]
@peastman any comments?

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+          - cuda: 10.2.89
+            gcc: 8.5.0
+            nvcc: 10.2
+            python: 3.8
+            pytorch: 1.7.1
+          - cuda: 11.2.2
+            gcc: 10.3.0
+            nvcc: 11.2
+            python: 3.9
+            pytorch: 1.8.0
```
Comment (peastman) [inline .github/workflows/ci.yml:24]: Should this be 1.9?  That's the most recent release.

Comment (raimis) [reply .github/workflows/ci.yml:24]: TorchANI has a problem with PyTorch 1.9.

**Comment by peastman** [comment]
Ok.  We just need to remember to update once that issue is fixed.  Everything else looks fine.

**Pull request merged**